From 28e28e9e598342c897ae5ca350d0da6f4aea057b Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 16 Jul 2021 16:02:36 +0200
Subject: [PATCH] build: Fix str/bool comparison

gnome-settings-daemon/meson.build:87: WARNING: Trying to compare values of different types (str, bool) using ==.
The result of this is undefined and will become a hard error in a future Meson release.
---
 meson.build | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/meson.build b/meson.build
index fdf71713..ba2a90ca 100644
--- a/meson.build
+++ b/meson.build
@@ -82,9 +82,7 @@ elif gsd_buildtype.contains('release')
   common_flags += ['-DG_DISABLE_CAST_CHECKS']
 endif
 
-# Workaround for meson's bug
-# https://github.com/mesonbuild/meson/pull/1896
-if get_option('b_ndebug') == true
+if get_option('b_ndebug') == 'true'
   common_flags += ['-DG_DISABLE_ASSERT']
 endif
 
-- 
GitLab

